### PR TITLE
New Port: Bugscraper

### PR DESCRIPTION
**Pull request by JanTrueno**
# New Port for Bugscraper

## Game Information
- **Title**: Bugscraper
- **URL**: https://github.com/Yolwoocle/bugscraper 

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [X] AmberELEC
- [X] ArkOS
- [X] JELOS

### Resolution Tests
Test all major resolutions:
- [X] 480x320
- [X] 640x480
- [X] 720x720 (RGB30)
- [X] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/bugscraper/port.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "inst": "This game is ready to run.",
+    "genres": [
+      "action"
+    ],
+    "image": null,
+    "rtr": true,
+    "exp": false,
+    "runtime": null,
+    "reqs": [],
+    "arch": []
```
Comment (kloptops) [inline ports/bugscraper/port.json:24]: No arch specified?

File: ports/bugscraper/bugscraper/log.txt
Comment (kloptops) [inline ports/bugscraper/bugscraper/log.txt:1]: Probably doesn't need to be commited. :)

**Review by kloptops — CHANGES_REQUESTED** [review]

**Review by Cebion — APPROVED** [review]
👍

**Review by kloptops — APPROVED** [review]
👍

**Pull request merged**